config/default.yml

```diff
@@ -3867,6 +3867,7 @@ Style/RedundantFreeze:
   Description: "Checks usages of Object#freeze on immutable objects."
   Enabled: true
   VersionAdded: '0.34'
+  VersionAdded: '0.66'
```

Comment:
Is it ok to have two `VersionAdded` or was this one supposed to be `VersionChanged`?

Reply:
It's supposed to be `VersionChanged `. Copy/paste error.